src/components/Slider/Slider.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import './Slider.css';
+import data from './dataSlider';
+import BtnSlider from './BtnSlider';
+import SliderCard from './SliderCard/SliderCard';
+
+export default function Slider () {
+
+    const picturesOnView = 4;
+    const [slideIndex, setSlideIndex] = useState(0)
+    const nextSlide = () => {
+        if (slideIndex < data.length - picturesOnView) {
+            setSlideIndex(slideIndex + 1)
+        };
+    };
+
+    const prevSlide = () => {
+        if(slideIndex > 0){
+            setSlideIndex(slideIndex - 1)
+        }
+    };
+
+    return (
+        <div className="container-slider">
+            {
+                data.slice(slideIndex, picturesOnView + slideIndex).map((obj, index) => {
+                return (
+                    <div className={"slide"} key = {index}>
```

Comment:
When you write string to props, you can remove `{}` and `key={index}` not `key = {index}`